easybuild/easyconfigs/g/GDCM/GDCM-2.8.7-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'GDCM'
+version = '2.8.7'
+
+homepage = 'http://gdcm.sourceforge.net/'
+description = """Grassroots DICOM (GDCM) is an implementation of the DICOM standard designed to be open source so
+that researchers may access clinical data directly. GDCM includes a file format definition and a network
+communications protocol, both of which should be extended to provide a full set of tools for a researcher
+or small medical imaging vendor to interface with an existing medical database."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [('http://sourceforge.net/projects/gdcm/files/%(namelower)s%202.x/%(name)s%20%(version)s', 'download')]
```

Comment:
Does using the `SOURCEFORGE_SOURCE` constant not work here? See https://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html#template-constants-that-can-be-used-in-easyconfigs

If not, maybe we should try and avoid hardcoding the major version in `202`, and use `20%(version_major)s` instead?
That will make the line too long though...